src/main/java/org/tarantool/TarantoolClientImpl.java

```diff
@@ -976,4 +1105,37 @@ public void close() {
 
     }
 
+    /**
+     * Used by internal services to ignore schema ID issues.
+     */
+    protected class UnsafeSchemaOps extends ClientDelegatingOps<List<?>> {
```

Comment:
It's too expensive to use an extra client, I think. I tried to reuse the existing one.

> It is not clear from the name that it performs synchronous operations. Are there chances that we'll need some ops class like that, but asynchronous?

It's for internal purposes only (for our background services, such as discovery or schema updater). Right now, I don't see any reasons to use an async version. Anyway, it can be refactored if circumstances require that.

Reply:
Should not it be package-pivate instead of public to make us sure we can change this API? Or there is some reason to make it public?

Reply:
`UnsafeSchemaOps` has protected visibility which a bit wider than package-private to be able to use these ops in derived clients in, possibly, different packages.